ui/src/main/webapp/components/CheckBoxComponent.stories.tsx

```diff
@@ -0,0 +1,38 @@
+import type { Meta, StoryObj } from '@storybook/react';
+import React, { useState } from 'react';
+import CheckBoxComponent from './CheckBoxComponent';
+
+const Meta = {
+    component: CheckBoxComponent,
+    title: 'Components/CheckBoxComponent',
+    render: (props) => {
+        // due to stories incompatibility, eslint rule is off
+        // React Hook "useState" is called in function "render" that is neither a React function component
+        const [state, setState] = useState(false); // eslint-disable-line react-hooks/rules-of-hooks
```

Comment:
I did not get why we need it. The component itself is a controlled component. Showing such stories may give a wrong perception that component has inner state